Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -774,7 +774,7 @@ void ApplicationManagerBase::InitAssetRequestHandler(AssetProcessor::AssetReques
         {
             if (!m_assetProcessorManagerIsReady)
             {
-                if (m_remainingAPMJobs == newNum)
+                if ((m_remainingAPMJobs == newNum) && m_remainingAPMJobs)
```

Comment:
This is the fix for skipping startup scan. Without this change, when we move to Idle, this signal is called with newNum == 0 and m_remainingAPMJobs never changed from zero. 

This is the sole place in the code that changes the state of  `m_assetProcessorManagerIsReady` which is one of the three things that must be true to signal back to the Editor that AP has completed enough work to allow launch to continue.